### DISCUSSION TITLE
Welcome to discord-giveaways Discussions!

### DISCUSSION THREAD

**Discussion by Androz2091** [Androz2091/discord-giveaways#381] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Nithhzzy** [comment]
it works or not if i change from node 16.9.0 to 18.20.3?
